jicofo/src/main/java/org/jitsi/jicofo/jigasi/TranscriberManager.java

```diff
@@ -275,5 +277,15 @@ public void memberPresenceChanged(@NotNull ChatRoomMember member)
         {
             TranscriberManager.this.memberPresenceChanged(member);
         }
+
+        @Override
+        public void transcriptionRequestedChanged(boolean transcriptionRequested)
+        {
+            if (transcriptionRequested)
```

Comment:
Shouldn't this check for the active flag too? Note that right now there will be 2 ways to trigger this and both will happen roughly at once:

- presence
- room metadata

Reply:
That's why the tasks are put on a single threaded executor. I can check the flag here, but I'll have to re-check it in the other thread too.

Reply:
👍  I'm not familiar with the code so I thought I'd ask. SGTM!

Reply:
It's a good question